assets/scss/layouts/products/_productSwatch.scss

```diff
@@ -12,8 +12,44 @@ $position_of_x : str-index($value_of_swatch_size, "x");
 $first_value : str-slice($value_of_swatch_size, 0, $position_of_x - 1);
 $second_value : str-slice($value_of_swatch_size, $position_of_x + 1);
 
+[data-product-attribute] .form-option.form-option-swatch {
```

Comment:
This selector is overriding the following selector: [data-product-attribute] .form-option